### PR TITLE
core: rename uses-webp-images to modern-image-formats

### PR DESCRIPTION
**Pull request by brendankenny**
part of #11866

With webp now being mainstream and AVIF and jpeg XL on their way, we're renaming the `uses-webp-images` audit to better reflect its goal, which is to suggest using ~~next-gen~~ \*ahem\* "modern" image formats (via e.g. `<picture>`), even if they're not available everywhere yet. See #12295 for details.

~~We didn't totally agree on this name yet, so we can decide here, which will be fun, and then I'll update https://github.com/GoogleChrome/lighthouse-stack-packs/pull/63 to match.~~ done

### CODE REVIEW

**Comment by connorjclark** [comment]
would prefer "(uses-)?modern-image-formats"

**Comment by brendankenny** [comment]
> would prefer "(uses-)?modern-image-formats"

"uses" is a no-go based on ["Naming: Audit ID"](https://github.com/GoogleChrome/lighthouse/blob/master/docs/new-audits.md#audit-id).

To me "modern" sounds a little more mainstream. Like I expect something that's part of the "modern web" to work on the majority of evergreen browsers. I don't feel strongly about modern vs next-gen, though.

File: lighthouse-core/audits/byte-efficiency/next-gen-image-formats.js [hunk 1]
```diff
@@ -25,13 +25,13 @@ const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
...
+class NextGenImageFormats extends ByteEfficiencyAudit {
   /**
    * @return {LH.Audit.Meta}
    */
   static get meta() {
     return {
-      id: 'uses-webp-images',
+      id: 'next-gen-image-formats',
       title: str_(UIStrings.title),
       description: str_(UIStrings.description),
```
Comment (adamraine) [inline lighthouse-core/audits/byte-efficiency/next-gen-image-formats.js:36]: We should probably update the docs to mention AVIF at some point. Maybe deprecate the WebP article and write a new one?

**Review by adamraine — COMMENTED** [review]
+1 to `modern-image-formats`

File: lighthouse-core/audits/byte-efficiency/next-gen-image-formats.js (hunk 1 above)
Comment (brendankenny) [reply lighthouse-core/audits/byte-efficiency/next-gen-image-formats.js:36]: > We should probably update the docs to mention AVIF at some point. Maybe deprecate the WebP article and write a new one?

👍 thats #12295

**Comment by brendankenny** [comment]
Note that the audit title has been "Serve images in next-gen formats" since Lighthouse 2.7.0, changed in #3985. This PR is only for the audit ID.

Humorously from #3985 and #3557 it appears @patrickhulce has long been in favor of "next gen" but @paulirish was against it, though later decided he was fine with the name :)

**Comment by brendankenny** [comment]
> https://www.thesaurus.com/browse/modern: `with-it-image-formats` 👀?

haha.

In https://github.com/GoogleChrome/lighthouse/pull/3985#discussion_r154825943 Paul suggested `2010-era` for the webp audit. AVIF support landed in Chrome 85, so I guess `2020-era-image-formats` would be accurate now and good for a few years before needing another rename :)

**Comment by patrickhulce** [comment]
> Humorously from #3985 and #3557 it appears @patrickhulce has long been in favor of "next gen" but @paulirish was against it, though later decided he was fine with the name :)

Haha, oooooh "superior" and "performant" -- hardworking, alphamale, jackhammer, merciless, insatiable image formats 😈 

I prefer `next-gen`, but I'm also fine with `modern` since it appears the majority has already spoken.

**Comment by brendankenny** [comment]
OK, unless he says otherwise, I'm going to assume @paulirish would also prefer `modern-image-formats` since he wrote "next-gen" _very_ reluctantly in #11866 and made sure to include the `modern` even though no one had said that out loud at the time :P

`modern-image-formats` it is

**Comment by brendankenny** [comment]
(all the stack-pack strings will get readded in the next dump->import cycle since only their name changed, not their content)

**Comment by addyosmani** [comment]
Fwiw, +1 on `modern-image-formats` :)

File: lighthouse-core/audits/byte-efficiency/modern-image-formats.js [hunk 2]
```diff
@@ -25,13 +25,13 @@ const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
 const IGNORE_THRESHOLD_IN_BYTES = 8192;
 
-class UsesWebPImages extends ByteEfficiencyAudit {
+class NextGenImageFormats extends ByteEfficiencyAudit {
```
Comment (adamraine) [inline lighthouse-core/audits/byte-efficiency/modern-image-formats.js:28]:
```suggestion
class ModernImageFormats extends ByteEfficiencyAudit {
```
and co

**Review by adamraine — CHANGES_REQUESTED** [review]

File: lighthouse-core/audits/byte-efficiency/modern-image-formats.js (hunk 2 above)
Comment (brendankenny) [reply lighthouse-core/audits/byte-efficiency/modern-image-formats.js:28]: whoops, thanks :)

**Review by adamraine — APPROVED** [review]

**Pull request merged**